gradle/scripts/dependencyDefinitions.gradle

```diff
@@ -47,7 +47,7 @@ ext.externalDependency = [
     "datanucleusCore": "org.datanucleus:datanucleus-core:3.2.10",
     "datanucleusRdbms": "org.datanucleus:datanucleus-rdbms:3.2.9",
     "eventhub": "com.microsoft.azure:azure-eventhubs:0.9.0",
-    "guava": "com.google.guava:guava:15.0",
+    "guava": "com.google.guava:guava:18.0",
```

Comment:
It's not a good time to upgrade guava to version 18, as the Hadoop API depends on guava APIs that are removed since guava 16.